docs/source/getting_started/03_custom_nodes.md

```diff
@@ -22,30 +22,23 @@ Once PeekingDuck has been installed, initialise a new project with our template
 > mkdir results
 ```
 
-### Folder Structure
-Your project folder should look like this at the end of the tutorial.
+Your project folder should look like this:
 
 ```bash
 .
 ├── src
 │   └── <custom_folder_name>
-│       ├── <node_type>               # <node_type> = "output" in this tutorial
-│       │    ├── utils
-│       │    │   └── csv.py
-│       │    └── csv_writer.py
 │       └── configs
-│            └── <node_type>          # <node_type> = "output" in this tutorial
-│                └── csv_writer.yml
 ├── run_config.yml
 ├── results
 ```
 
 - `src/<custom_folder_name>` is where custom nodes created for the project should be housed.
 - `run_config.yml` is the basic yaml file to select nodes in the pipeline. You will be using this to run your peekingduck pipeline.
 
-## Step 2: Populate run_config
+## Step 2: Populate run_config.yml
 
-The `run_config.yml` will be created. Input the following lines of code into the file. Newly minted nodes needs to be identified by the following structure `<custom_folder_name>.<node-type>.<node>`. In this project, the `<custom_folder_name>` is named `custom_nodes`.
+The `run_config.yml` will be created by `peekingduck init`. For this tutorial, replace its contents with the following node list. Notice that newly minted nodes are identified by the following structure `<custom_folder_name>.<node-type>.<node>`. In this project, the `<custom_folder_name>` is named `custom_nodes`.
```

Comment:
node-type comprise of 5 types...
either link to the diagram in RTD intro or just list them here